showcase/tests/integration/components/hds/time/index-test.js

```diff
@@ -0,0 +1,282 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test, skip } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | hds/time/index', function (hooks) {
+  setupRenderingTest(hooks);
+
+  let service;
+
+  hooks.beforeEach(function () {
+    service = this.owner.lookup('service:hdsTime');
+  });
+
+  test('it should render the component with a CSS class that matches the component name', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:48" id="test-time" />
+    `);
+    assert.dom('#test-time').hasClass('hds-time');
+  });
+
+  // OPTIONS
+
+  // OPTION: hasTooltip
+
+  test('it should render the component with a tooltip by default', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:48" />
+    `);
+    assert.dom('.hds-tooltip-button').exists();
+  });
+
+  test('it should render the component without a tooltip', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:48" @hasTooltip={{false}} />
+    `);
+    assert.dom('.hds-tooltip-button').doesNotExist();
+  });
+
+  // OPTION: isOpen (tooltip)
+
+  test('it should render the component with the tooltip closed by default', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:48" />
+    `);
+    assert.dom('.tippy-box').doesNotExist();
+  });
+
+  test('it should render the component with the tooltip open', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:48" @isOpen={{true}} />
+    `);
+    assert.dom('.tippy-box').exists();
+  });
+
+  // OPTION: display
+
+  // Default display type
+
+  // default using Date object
+  test('it should render the correct string in the default format when passing a Date object', async function (assert) {
+    this.set('defaultDate', new Date('05 September 2018 14:07:32'));
+    await render(hbs`
+      <Hds::Time @date={{this.defaultDate}} @isOpen={{true}} />
+    `);
+    assert.dom('.hds-time').hasText('Sep 5, 2018, 2:07:32 PM PDT');
```

Comment:
Tried adding time zones to friendly local times but the tests fail when pushed up because the server is in a different time zone.

Reply:
Fixed these by creating the expected date string using Date object.